Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml

```diff
@@ -2,36 +2,91 @@
+            <div id="@Model.GetFieldName("TextInput-info")" class="govuk-hint govuk-character-count__message">
+                @string.Format(StaticTextResource.GovukFrontend_CharacterCount_CharactersAvailable, "10,000")
+            </div>
+        </div>
     }
+}
 
-    @if (hasError)
+<div class="govuk-form-group @((Model.IsRequired && textInputHasError) || selectOptionHasError ? "govuk-form-group--error" : "")">
+    @if (Model.IsRequired)
     {
-        <p class="govuk-error-message" id="@Model.GetFieldName("TextInput-error")">
-            <span class="govuk-visually-hidden">Error:</span>
-            @Model.GetValidationMessage(nameof(Model.TextInput), ViewData.ModelState)
-        </p>
+        @RenderLabelHeading(Model.Heading, Model.GetFieldName(nameof(Model.TextInput)), Model.Caption, Model.IsFirstQuestion, Model.Options?.Layout?.Heading?.Size)
+        RenderDescription();
+        RenderField();
     }
+    else
+    {
+        <fieldset class="govuk-fieldset" @(selectOptionHasError ? Html.Raw($"aria-describedby=\"{Model.GetFieldName("input-error")}\"") : "")>
+            @RenderLegendWithoutCaption(Model.Heading, Model.IsFirstQuestion, Model.Options?.Layout?.Heading?.Size)
+            @{
+                RenderDescription();
+            }
+            @if (selectOptionHasError)
+            {
+                <p class="govuk-error-message" id="@Model.GetFieldName("input-error")">
+                    <span class="govuk-visually-hidden">@StaticTextResource.Global_Error:</span>
+                    @Model.GetValidationMessage(nameof(Model.HasValue), ViewData.ModelState)
+                </p>
+            }
+            <div class="govuk-radios" data-module="govuk-radios">
+
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="@Model.GetFieldName(nameof(Model.HasValue))" name="@Model.GetFieldName(nameof(Model.HasValue))" type="radio"
+                           value="true" data-aria-controls="@Model.GetFieldName("conditional-YES")" @(Model.HasValue == true ? "checked" : "")>
+                    <label class="govuk-label govuk-radios__label" for="@Model.GetFieldName(nameof(Model.HasValue))">@StaticTextResource.Global_Yes</label>
+                </div>
 
-    <textarea class="govuk-textarea govuk-js-character-count @(Model.Options?.Layout?.Input?.CustomCssClasses)" rows="5" id="@Model.GetFieldName(nameof(Model.TextInput))" name="@Model.GetFieldName(nameof(Model.TextInput))" aria-describedby="@(hasError ? Model.GetFieldName("TextInput-error") + " " : "")@Model.GetFieldName("TextInput-info")">@Model.TextInput</textarea>
+                <div class="govuk-radios__conditional" id="@Model.GetFieldName("conditional-YES")">
+                    <div class="govuk-form-group @(textInputHasError ? "govuk-form-group--error" : "")">
+                        @{
+                            <label class="govuk-label" for="@nameof(Model.TextInput)">@Model.Caption</label>
```

Comment:
I think the 'for' value should be @Model.GetFieldName(nameof(Model.TextInput)) to account for multiple questions, I think URL Input also has this wrong